tools/tiller-hack/tiller-hack.go

```diff
@@ -0,0 +1,232 @@
+package main
```

Comment:
It would be useful to describe at the top what this code is doing at a high level step by step.
Also an example usage would be good.

Reply:
done